content/doc/book/system-administration/monitoring.adoc

```diff
@@ -35,3 +35,59 @@ See the jira:WEBSITE-738[] EPIC for tasks related to this page, contributions ar
 - plugin:versioncolumn[Versions Node Monitors]
 - plugin:systemloadaverage-monitor[Agents Monitoring for unix nodes]
 - plugin:jqs-monitoring[Job/Queue/Slaves Monitoring]
+
+
+== Ping Thread
+
+Jenkins installs "ping thread" on every remoting connection, such as controller/agent connections, regardless of its
+transport mechanism (such as SSH, JNLP, etc.) The lower level of the Jenkins remoting protocol is a message oriented
```

Comment:
`transport mechanism (such as SSH, JNLP, etc.)` is missing a `.`

Reply:
updated directly